docs/dependency-injection.md

```diff
@@ -54,7 +54,7 @@ Services can be configured with the following lifetimes:
 | ----------- | ----------- |
 | [Singleton](https://docs.microsoft.com/dotnet/api/microsoft.extensions.dependencyinjection.servicedescriptor.singleton#Microsoft_Extensions_DependencyInjection_ServiceDescriptor_Singleton__1_System_Func_System_IServiceProvider___0__) | DI creates a *single instance* of the service. All components requiring this service receive a reference to this instance. |
 | [Transient](https://docs.microsoft.com/dotnet/api/microsoft.extensions.dependencyinjection.servicedescriptor.transient) | Whenever a component requires this service, it receives a *new instance* of the service. |
-| [Scoped](https://docs.microsoft.com/dotnet/api/microsoft.extensions.dependencyinjection.servicedescriptor.scoped) | Blazor doesn't currently have the concept of DI scopes. `Scoped` behaves like `Singleton`. Therefore, prefer `Singleton` and avoid `Scoped`. |
+| [Scoped](https://docs.microsoft.com/dotnet/api/microsoft.extensions.dependencyinjection.servicedescriptor.scoped) | Client-side Blazor doesn't currently have the concept of DI scopes. `Scoped` behaves like `Singleton`. Therefore, prefer `Singleton` and avoid `Scoped` in client-side Blazor apps. However, server-side Blazor, also known as Razor Components, supports the `Scoped` lifetime. In a server-side Razor Component, a scoped service registration is scoped to the request. |
```

Comment:
I'm not sure what "to the request" means in this context, since everything in serve-side Blazor is happening over a SignalR connection. @bradygaster do you have a recommendation here?

Reply:
@danroth27 by "to the request" i presume what's meant is "to the HTTP request" but in the case of Blazor - not sure. Or is your question more about what the scope should be for SignalR hubs being used by server-side Blazor?

Reply:
A comment in the Hubs topic states ...

> Hubs are transient ... Every hub method call is executed on a new hub instance.

... every Blazor SignalR call receives a fresh service instance for any service registered as scoped?

Reply:
if I'm not mistaken: from a user point of view, a scoped instance is created for every browser tab a user opens  thus for every SingalR connection. A singleton  on the otherhand is  shared between all tabs for all connected users.

Reply:
Yeah, that sounds right. So I think this should say "to the connection" instead of "to the request". Otherwise, looks good to me.